conda_smithy/configure_feedstock.py

```diff
@@ -319,6 +322,7 @@ def main(forge_file_directory):
     recipe_dir = 'recipe'
     config = {'docker': {'image': 'condaforge/linux-anvil', 'command': 'bash'},
               'templates': {'run_docker_build': 'run_docker_build_matrix.tmpl'},
+              'ci_enabled': [],
```

Comment:
Have added code that strips `<CI>.enabled` from the `config`. If it find `<CI>.enabled` for any CI, it issues a warning that explains how to configure `skip`.